scaffolder-templates/gitops/bootstrap/README.md

```diff
@@ -7,11 +7,14 @@ Before applying the manifests, there is a need to replace the `__REPLACE_SSH_PRI
 Either manually edit the file or login to the target cluster and run the following command:
 
 ```
+# Change the Git host to the one you are using, for example, github.com, gitlab.cee.redhat.com, gitlab.gitlab, etc.
+
 git clone https://github.com/${{ values.orgName }}/${{ values.repoName }}.git
 cd ${{ values.repoName }}/bootstrap
 
 # Optionally, if edited the file manually
-SSH_PRIVATE_KEY=$(oc get secrets -n orchestrator-gitops git-ssh-credentials -o jsonpath='{.data.id_rsa}')
+SSH_PRIVATE_KEY=$(oc get secrets -n orchestrator-gitops git-ssh-credentials -o jsonpath='{.data.id_rsa}') 
+# For a gitlab client, use gitlab-ssh-credentials instead. 
```

Comment:
The comment should be above the command no?

But more important, is it possible to add an `if` statement so when the template is evaluated, the right name will be displayed right away?

Reply:
Honestly, I couldn't find a way to add a conditional to a PipelineRun template. I tried inserting a bash script but it wouldn't render in runtime. I guess no flow control is allowed as the pipelinerun template was expecting static vars. 
Another option was to add a variable like 'values.gitCredentialsHost' and use that in the template, but that would have included more logic in the templates themselves (and changes to the github templates also). 
I opted for changing the creds dynamically on gitlab

Reply:
>  I couldn't find a way to add a conditional to a PipelineRun template
mmmh, that's annoying, you tried something like
```
{% if values.gitlab %}
SSH_PRIVATE_KEY=$(oc get secrets -n orchestrator-gitops gitlab-ssh-credentials -o jsonpath='{.data.id_rsa}') 
{% endif %}
``` 
in the README and it was not interpreted when fetching the template? (https://backstage.io/docs/features/software-templates/writing-templates/#the-templating-syntax)

On a side note, maybe renaming  `git-ssh-credentials` to `github-ssh-credentials` would make sense, consistency wise